Userland/Utilities/headless-browser.cpp

```diff
@@ -412,7 +412,7 @@ static ErrorOr<TestResult> run_ref_test(HeadlessWebContentView& view, StringView
     VERIFY(actual_screenshot);
     VERIFY(expectation_screenshot);
 
-    if (actual_screenshot->visually_equals(*expectation_screenshot))
+    if (actual_screenshot->visually_equals(*expectation_screenshot, 128))
         return TestResult::Pass;
```

Comment:
ok, I changed the test to use rectangle mask instead circle mask so now this is no longer needed.

I think it is good to avoid introducing threshold for as long as we can, but WPT has 256px threshold, as I recall, on ref-tests so it is probably fine.

Reply:
changed the test back to use circles, but now comparing svg circle (mask) against svg circle (non-mask) so they match.